src/category_theory/sites/types.lean

```diff
@@ -0,0 +1,94 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+
+import category_theory.sites.sheaf
+
+/-!
+# Grothendieck Topology and Sheaves on the Category of Types
+
+In this file we define the grothendieck topology on the category of types,
```

Comment:
There are other interesting topologies on Type besides this one, so could we rephrase this to `a Grothendieck topology` or something like that?

Reply:
Done.